warp10/src/main/java/io/warp10/script/functions/GET.java

```diff
@@ -85,4 +100,22 @@ public static int computeAndCheckIndex(NamedWarpScriptFunction func, int index,
 
     return index;
   }
+
+  static Object recNestedGet(NamedWarpScriptFunction func, List nestedList, List<Number> indexList) throws WarpScriptException {
```

Comment:
This method should not need the func attribute, the WarpScriptException should be caught by the caller which will wrap it and add its name

Reply:
I think it needs the func attribute in case it is used by another function to have a better error message.